README.md

```diff
@@ -7,9 +7,33 @@ This project is a basic local server for the [Riff Racer](https://store.steampow
 - Python 3.6+
 - Flask
 
+(Or Docker)
+
 ## Usage
 
-Run `sed -i "s/https:\/\/parseapi.back4app.com/http:\/\/127.000000000.0.1:5144/g" ParseOctane.dll` in your game directory to patch the Parse API DLL to use RiffApi Local instead. Then, run `riffapi-local.py` and launch the game.
+### Patching ParseOctane.dll
+
+Patch `ParseOctane.dll` to use RiffApi Local changing "https://parseapi.back4app.com" by "http://127.000000000.0.1:5144".
+
+1. Open Steam and go to your game library
+2. Right click on "Riff Racer" on the game list
+3. Press on "manage" > "browse local files", it will open the folder where the game files are located.
+4. Open a command prompt, `shift` + `right-click` and "Open PowerShell window here
+5. Run `sed -i "s/https:\/\/parseapi.back4app.com/http:\/\/127.000000000.0.1:5144/g" ParseOctane.dll` or open "ParseOctane.dll" with a text editor and replace all the occurrences of "https://parseapi.back4app.com" by "http://127.000000000.0.1:5144".
+
+### Running the API
+
+You have two options:
+
+1. Use [Docker](https://www.docker.com/products/docker-desktop/) (recommended).
```

Comment:
Imo just plain python should be the first and recommended option, not everything needs docker, especially a simple script like this

Reply:
I think that the people that will be using this tool will know little about what they are doing. Docker guarantees it will work the same in any computer I chose to set Docker as the "recommended" option.

Else I don't really care what option is set as "recommended".

Reply:
I'd still rather Docker be the secondary option.

Reply:
Ok, I'll set it up as the second one